main.py

```diff
@@ -90,7 +90,15 @@ def main():
     gac = GACAgent(args)
 
     state = env.reset()
-
+    
+    results_dict = {'train_rewards': []
+                    'eval_rewards': [],
+                    'actor_losses': [],
+                    'value_losses': [],
+                    'policy_losses': [],
```

Comment:
I think policy_losses really means critic_losses?

Reply:
Yes, it was typo. I'll fix it.

Reply:
fixed